mailpoet/tests/docker/docker-compose.yml

```diff
@@ -104,6 +104,8 @@ services:
       MYSQL_PASSWORD: wordpress
     volumes:
       - /dev/shm:/dev/shm
+    ports:
```

Comment:
Could you please elaborate on why this is needed and why in this PR? Asking as it is not mentioned anywhere in the PR description or the Jira ticket.